src/main/kotlin/com/github/warningimhack3r/intellijshadcnplugin/backend/helpers/DependencyManager.kt

```diff
@@ -35,12 +35,22 @@ class DependencyManager(private val project: Project) {
         }.values.firstOrNull()
     }
 
+    private fun getInstallCommand(packageManager: String): String {
+        return when (packageManager) {
+            "npm" -> "i"
+            "pnpm" -> "add"
+            "yarn" -> "add"
```

Comment:
I'd leave `i` for every package manager other than yarn, as it's supported with all of them
It would shorten the code even more: `add` if yarn, else `i`! ;)